global_helpers/panther_base_helpers.py

```diff
@@ -139,6 +140,65 @@ def gsuite_details_lookup(detail_type, detail_names, event):
     return {}
 
 
+# # # # # # # # # # # # # #
+#      Zendesk Helpers     #
+# # # # # # # # # # # # # #
+ZENDESK_APP_ROLE_ASSIGNED = re.compile(
+    r"(?P<app>.*) role changed from (?P<old_role>.+) to (?P<new_role>.*)", re.IGNORECASE
+)
```

Comment:
Let's see how many of these can be moved to live in the datamodel.

Reply:
Moved all but the role-related ones (which is used in 2 rules)